teaspoons-cli/tests/commands/test_pipelines_commands.py

```diff
@@ -0,0 +1,105 @@
+# tests/commands/pipelines_tests.py
+
+import logging
+from click.testing import CliRunner
+from mockito import when, verify
+from teaspoons.commands import pipelines_commands
+from teaspoons_client.models.pipeline import Pipeline
+from teaspoons_client.models.pipeline_with_details import PipelineWithDetails
+from teaspoons_client.exceptions import ApiException
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+def test_list_pipelines(caplog):
+    runner = CliRunner()
+    test_pipelines = [
+        Pipeline(
+            pipeline_name="test_pipeline_1",
+            display_name="test_display_name_1",
+            description="test_description_1",
+        ),
+        Pipeline(
+            pipeline_name="test_pipeline_2",
+            display_name="test_display_name_2",
+            description="test_description_2",
+        ),
+    ]
+
+    when(pipelines_commands.pipelines_logic).list_pipelines().thenReturn(test_pipelines)
+
+    with caplog.at_level(logging.DEBUG):
+        result = runner.invoke(pipelines_commands.pipelines, ["list"])
+
+    assert result.exit_code == 0
+    verify(pipelines_commands.pipelines_logic).list_pipelines()
+    assert "Found 2 available pipelines:" in caplog.text
+    assert "test_pipeline_1" in caplog.text
+    assert "test_pipeline_2" in caplog.text
+
+
+def test_get_info_success(caplog, unstub):
+    runner = CliRunner()
+    test_pipeline = PipelineWithDetails(
+        pipeline_name="test_pipeline",
+        description="test_description",
+        display_name="test_display_name",
+        type="test_type",
+        inputs=[],
+    )
+
+    when(pipelines_commands.pipelines_logic).get_pipeline_info(
+        "test_pipeline"
+    ).thenReturn(test_pipeline)
+
+    with caplog.at_level(logging.DEBUG):
+        result = runner.invoke(
+            pipelines_commands.pipelines, ["get-info", "test_pipeline"]
+        )
+
+    assert result.exit_code == 0
+    verify(pipelines_commands.pipelines_logic).get_pipeline_info("test_pipeline")
+    assert "test_pipeline" in caplog.text
```

Comment:
is there any way we can test the full string in some of these tests to guard against the issue we were seeing earlier where the logger wasnt properly set up in this testing suite.

Or even just a test that logs something and makes sure its formatted correctly

Reply:
i don't really care about the test logger being formatted correctly. i think if we want to test that the logs are formatted correctly we should add a test of `cli.py` (where the log formatting is set)